qsr_lib/cfg/defaults.ini

```diff
@@ -1,2 +1,3 @@
 [arg_relations_distance]
-relations_and_values = {"0": 5., "1": 15., "2": 120.}
+;relations_and_values = {"0": 5., "1": 15., "2": 120.}
+relations_and_values = {}
```

Comment:
But having an optional config file with an empty dictionary doesn't make sense. Just don't use this config file then.
Btw. what happens if you give it a config file but doesn't contain an entry for the specific QSR you want? I'm sure it just ignores it?